src/Membership/MemberActivityCollection.php

```diff
@@ -18,10 +18,13 @@ class MemberActivityCollection
      */
     public function __construct(Adherent $adherent, array $eventRegistrations)
     {
-        $this->activities[self::formatDate($adherent->getActivatedAt())][] = [
-            'type' => self::TYPE_JOINED_EN_MARCHE,
-            'log' => sprintf('A rejoint le mouvement En Marche (%s)', self::formatDate($adherent->getActivatedAt())),
-        ];
+        $this->activities = [];
+        if ($adherent->getRegisteredAt()) {
```

Comment:
We imported data from the past, we may have users without registeredAt. Let's assume this can be the case.